.github/workflows/approved-for-ci-run.yml

```
@@ -93,6 +93,7 @@ jobs:
                                                        --body-file "body.md" \
                                                        --head "${BRANCH}" \
                                                        --base "main" \
+                                                       --label "run-e2e-test-in-draft" \
```

Let's rename the label:
```suggestion
                                                       --label "run-e2e-tests-in-draft" \
```

---

sure